CONTRIBUTING.md

```
@@ -0,0 +1,22 @@
+We recommend following these steps if you'd like to contribute to the project.
+
+1. Fork the yapp repository.
+2. Develop on a [feature branch](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
+3. Submit a pull request (don't review your own)!
+
+### Reporting bugs
+
+If you found a bug that is not a security issue, file an issue in GitHub.
+
+If you found a bug that is a security issue, do not file an issue in Github.
+Instead, either file a pull request with the fix, or email dthaler1968@gmail.com.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)